Content.Shared/Damage/Systems/DamageableSystem.cs

```diff
@@ -195,12 +197,26 @@ public void SetDamageModifierSetId(EntityUid uid, string damageModifierSetId, Da
 
             Dirty(comp);
         }
-        
+
         private void DamageableGetState(EntityUid uid, DamageableComponent component, ref ComponentGetState args)
         {
             args.State = new DamageableComponentState(component.Damage.DamageDict, component.DamageModifierSetId);
         }
 
+        private void OnIrradiated(EntityUid uid, DamageableComponent component, OnIrradiatedEvent args)
+        {
+            var damageValue = FixedPoint2.New(MathF.Max((args.TotalRads), 1));
```

Comment:
IIRC the `Max(... , 1)` was from before damage was moved from ints to `FixedPoint2`. Probably fine to leave it for now, but the rounding should probably be reconsidered, so that smaller radiation values don't round up.

Reply:
I replaced it with:
```
var damageValue = FixedPoint2.New(args.TotalRads);
```